frontends/web/src/routes/lightning/send/send.tsx

```diff
@@ -46,7 +46,138 @@ const SendingSpinner = () => {
   return <Spinner text={message} guideExists={false} />;
 };
 
-export function Send() {
+type InvoiceInputProps = {
+  invoice: LnInvoice;
+};
+
+const InvoiceInput = ({ invoice }: InvoiceInputProps) => {
+  const { t } = useTranslation();
+  const balance: accountApi.IBalance = {
+    hasAvailable: true,
+    available: {
+      amount: `${toSat(invoice.amountMsat || 0)}`,
+      // TODO: conversions missing
+      unit: 'sat'
+    },
+    hasIncoming: false,
+    incoming: {
+      amount: '0',
+      unit: 'sat'
+    }
+  };
+  return (
+    <>
+      <h1 className={styles.title}>{t('lightning.send.confirm.title')}</h1>
+      <div className={styles.info}>
+        <h2 className={styles.label}>{t('lightning.send.confirm.amount')}</h2>
+        <Amount amount={balance.available.amount} unit={balance.available.unit} removeBtcTrailingZeroes />/{' '}
+        <FiatConversion amount={balance.available} noBtcZeroes />
+      </div>
+      {invoice.description && (
+        <div className={styles.info}>
+          <h2 className={styles.label}>{t('lightning.send.confirm.memo')}</h2>
+          {invoice.description}
+        </div>
+      )}
+    </>
+  );
+};
+
+type PaymentInputProps = {
+  input: InputType;
+};
+
+const PaymentInput = ({ input }: PaymentInputProps) => {
+  switch (input.type) {
+  case InputTypeVariant.BOLT11:
+    return (
+      <InvoiceInput invoice={input.invoice} />
+    );
+  }
+};
+
+type SendWorkflowProps = {
+  onBack: () => void;
+  onInput: (input: string) => void;
+  onSend: () => void;
+  parsedInput?: InputType;
+  rawInputError?: string;
+  step: TStep;
+};
+
+const SendWorkflow = ({
+  onBack,
+  onInput,
+  onSend,
+  parsedInput,
+  rawInputError,
+  step,
+}: SendWorkflowProps) => {
+  const { t } = useTranslation();
+  switch (step) {
+  case 'select-invoice':
+    return (
+      <View fitContent>
+        <ViewContent textAlign="center">
+          <Grid col="1">
+            <Column>
+              {/* this flickers quickly, as there is 'SdkError: Generic: Breez SDK error: Unrecognized input type' when logging rawInputError */}
```

Comment:
@dangeross what does this comment mean?

Reply:
I think @thisconnect added this comment. The `rawInputError` is the result of calling `parseInput()` (called by onResult of the ScanQRVideo component) with an invalid input type. It's looking for BOLT11, LN-address, LNURL etc. Could be it's called with an empty string?

Reply:
I have to check this again and would like to keep the comment for now.

Reply:
oh ok! Sorry @dangeross